ipn/ipnlocal/local.go

```diff
@@ -2133,6 +2133,20 @@ func (b *LocalBackend) DebugNotify(n ipn.Notify) {
 	b.send(n)
 }
 
+// DebugForceNetmapUpdate forces a full no-op netmap update
```

Comment:
Sentence can be reworded a bit to explain that it reloads the netmap in different systems